war/src/main/java/io/lumeer/engine/controller/CollectionFacade.java

```diff
@@ -90,309 +78,226 @@
     *
+    * @param collectionCode
+    *       collection code
     * @param collectionName
-    *       the internal name of the collection where the given attribute should be removed
+    *       collection name
     * @param attributeName
     *       name of the attribute to remove
-    * @throws CollectionNotFoundException
-    *       if collection was not found in database
     */
-   public void dropAttribute(final String collectionName, final String attributeName) throws CollectionNotFoundException {
-      if (dataStorage.hasCollection(collectionName)) {
-         collectionMetadataFacade.dropAttribute(collectionName, attributeName);
-         List<DataDocument> documents = getAllDocuments(collectionName);
+   public void dropAttribute(final String collectionCode, final String collectionName, final String attributeName) {
+      collectionMetadataFacade.dropAttribute(collectionCode, attributeName);
+      List<DataDocument> documents = getAllDocuments(collectionName);
 
-         for (DataDocument document : documents) {
-            dataStorage.dropAttribute(collectionName, dataStorageDialect.documentIdFilter(document.getId()), attributeName);
-         }
-
-         collectionMetadataFacade.setLastTimeUsedNow(collectionName);
-      } else {
-         throw new CollectionNotFoundException(ErrorMessageBuilder.collectionNotFoundString(collectionName));
+      for (DataDocument document : documents) {
+         dataStorage.dropAttribute(collectionName, dataStorageDialect.documentIdFilter(document.getId()), attributeName);
       }
+
+      collectionMetadataFacade.setLastTimeUsedNow(collectionCode);
    }
 
    /**
     * Updates the name of an attribute in all documents of given collection.
     *
+    * @param collectionCode
+    *       collection code
     * @param collectionName
-    *       internal name of the collection where the given attribute should be renamed
+    *       collection name
     * @param origName
     *       old name of an attribute
     * @param newName
     *       new name of an attribute
-    * @throws CollectionNotFoundException
-    *       if collection was not found in database
     * @throws AttributeAlreadyExistsException
     *       when attribute with new name already exists
     */
-   public void renameAttribute(final String collectionName, final String origName, final String newName) throws CollectionNotFoundException, AttributeAlreadyExistsException {
-      if (dataStorage.hasCollection(collectionName)) {
-         collectionMetadataFacade.renameAttribute(collectionName, origName, newName);
-         dataStorage.renameAttribute(collectionName, origName, newName);
+   public void renameAttribute(final String collectionCode, final String collectionName, final String origName, final String newName) throws AttributeAlreadyExistsException {
+      collectionMetadataFacade.renameAttribute(collectionCode, origName, newName);
+      dataStorage.renameAttribute(collectionName, origName, newName);
```

Comment:
Why do we pass `collectionName` here? Should not collections be identified by `collectionCode` only?